### PR TITLE
feat: alternative routes example

### PR DESCRIPTION
**Pull request by lsmienk**
This example shows alternative routes as well as the fasted routes. You can select one of the alternatives, that route will be highlighted and its journey specs will be displayed :).

### CODE REVIEW

File: alternative-routes/map.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      data: {
+        type: 'FeatureCollection',
+        features: points,
+      },
+    },
+  });
+};
+
+/**
+ * Highlight the route that was clicked.
```
Comment (Khrystyna) [inline alternative-routes/map.js:185]: It says "highlight", but the name of the method says "change" If this method indeed highlights the route, I would then use it in the name of the method.

File: alternative-routes/map.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    });
+
+    map.on('mouseleave', `${i}`, () => {
+      map.getCanvas().style.cursor = '';
+      popup.remove();
+    });
+  }
+
+  const routeOptions = document.querySelectorAll('input[type=radio][name="routes"]');
+  routeOptions.forEach((route, index) => route.addEventListener('change', () => changeRoute(routes, index)));
```
Comment (Khrystyna) [inline alternative-routes/map.js:68]: looks like we can just pass here `routes[index]` and then no need to pass the entire routes array?  And I would also switch position around meaning first pass id and then the object. Usually it's done in this order :)

File: public/alternative-routes/index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        <div class="tags">
+          <div id="distance" class="tag"></div>
+          <div id="stop" class="tag"></div>
+          <div id="consumption-overview" class="tag"></div>
+        </div>
+      </header>
+
+      <section>
+        <div class="route-selector" id="selector">
+          <input class="checkbox-routes" type="radio" name="routes" id="route-0" checked />
```
Comment (Khrystyna) [inline public/alternative-routes/index.html:38]: Very nice that you grouped them together via 'name' property! Then in JS it gives you an advantage ;)

**Review by Khrystyna — APPROVED** [review]

File: alternative-routes/map.js (hunk 2 above)
Comment (lsmienk) [reply alternative-routes/map.js:68]: I am passing through all routes because we need to change the color to grey for the routes that are not selected.

**Pull request merged**